shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/components/Tag.kt

```diff
@@ -46,17 +56,20 @@ fun Tag(
     }
 }
 
+@OptIn(ExperimentalResourceApi::class)
 @Composable
 fun LightningTalk(title: String, dimmed: Boolean = false) {
-    Tag(Icons.LIGHT, title, dimmed)
+    Tag(Res.drawable.light.painter(), title, dimmed)
```

Comment:
here and in other places I see you use a painter as a function parameter.
I would say that to pass a drawable is better and get painter in the function inside

Reply:
refactored